apps/server/src/models/transactions.ts

```diff
@@ -15,16 +16,17 @@ export enum action {
   CREDIT = 'Credit',
 }
 
-export enum transaction_status {
+export enum transactionStatus {
   SUCCESSFUL = 'Successful',
   PENDING = 'Pending',
   UNSUCCESSFUL = 'Unsuccessful',
 }
 
-export enum transaction_type {
-  GLOBAL = 'Global',
-  SAVINGS = 'Savings',
-  GROUP_WALLET = 'Group Wallet',
+export enum transactionType {
+  GLOBAL_TRANSACTIONS = 'Global fransactions', //Transactions between global wallet and external source/destination
+  SAVINGS_TRANSACTIONS = 'Savings transcations', //Transactions between personal savings wallet and all goals wallets.
+  GROUP_TRANSACTIONS = 'Group Wallet Transactions', //Transactions between personal group wallet and all groups wallets.
+  INTERNAL_TRANSFER = 'Internal funding', // Transactions between global wallet and other personal wallets(savings, group etc.)
 }
```

Comment:
I really want us to stop using enum. While it's not originally part of javascript, it has downsides that we should avoid 
eg: repetition

Reply:
prefer unions

type transactionStatus = 'successful' | 'peding' | 'unsuccesful'

Reply:
The spelling of fransactions should be corrected please.
I tap from your ocean of knowledge, boss.

Reply:
Changing the enums means we have to have use unions in all the models, then?